nevergrad/optimization/test_doc.py

```diff
@@ -128,6 +128,7 @@ def myfunction(arg1, arg2, arg3, amount=(2, 2)):
 
 
 def test_doc_constrained_optimization() -> None:
+    np.random.seed(12)  # avoid flakiness
```

Comment:
actually this test seems to be seeded from somewhere and reproducibly crashes without this (but not on windows) which is very concerning